lib/main.dart

```diff
@@ -52,7 +54,7 @@ class MyApp extends StatelessWidget {
     if (kIsWeb) {
       // if width is less than 530, it means the user resized the window to a smaller size
       if (MediaQuery.of(context).size.width < 530) {
-        designSize = const Size(390, 844);
+        designSize = const Size(1390, 844);
```

Comment:
This design size is for mobile, it shouldn't be changed otherwise it will break on mobile